barretenberg/cpp/src/barretenberg/crypto/merkle_tree/node_store/cached_content_addressed_tree_store.hpp

```diff
@@ -941,14 +969,22 @@ void ContentAddressedCachedTreeStore<LeafValueType>::remove_historical_block(con
         get_meta(uncommittedMeta, *tx, true);
         get_meta(committedMeta, *tx, false);
         if (blockNumber != committedMeta.oldestHistoricBlock) {
-            throw std::runtime_error("Block number is not the most historic");
+            throw std::runtime_error(
```

Comment:
This formatting could have a small utility func

Reply:
Woops sorry hit merge by accident

Reply:
Used the pre-existing `format` function